src/smdba/postgresqlgate.py

```diff
@@ -137,6 +137,8 @@ def estimate(self):
         """
         Estimate the data.
         """
+
+        PG_VERSION = [int(v_el) for v_el in os.popen("psql --version | sed -e 's/.*\s//g'").read().split('.')]
```

Comment:
No, sure you can write as you wish, but PEP and conventions are still valid. Same as you do not use tabs in bash scripting. Not because bash is dictating, but because people will hateya. :)